test/fixtures/test_logger.js

```diff
@@ -19,6 +19,13 @@ class TestLogger {
     return true;
   }
 
+  clear() {
```

Comment:
Ah scratch that, I see that it needs to be cleaned because `data.getAll()` will fill the logger.

Reply:
Thanks for this as i was going to implement it for my tests